cpp/src/parquet/encoding_benchmark.cc

```diff
@@ -25,6 +25,7 @@
 #include "arrow/testing/util.h"
 #include "arrow/type.h"
 #include "arrow/util/byte_stream_split.h"
+#include "arrow/util/cpu_info.h"
```

Comment:
Why did you need this by the way?

Reply:
The size of test spaced block was set to L1 size. 
arrow::internal::CpuInfo::GetInstance()->CacheSize(arrow::internal::CpuInfo::L1_CACHE);

Reply:
Using a hardcoded size would be just as good (or even better, to compare between machines).

Reply:
Done